scripts/run-pyright.py

```diff
@@ -200,16 +230,24 @@ def normalize_env(env: str, rebuild: bool) -> None:
             [
                 f"python -m venv {venv_path}",
                 f"{venv_path}/bin/pip install -U pip setuptools wheel",
-                (
-                    f"{venv_path}/bin/pip install -r"
-                    # find-links for M1 lookup of grpcio wheels
-                    f" {requirements_path} --find-links=https://github.com/dagster-io/build-grpcio/wiki/Wheels"
+                " ".join(
+                    [
+                        f"{venv_path}/bin/pip",
+                        "install",
+                        "-r",
+                        requirements_path,
+                        *EXTRA_PIP_INSTALL_ARGS,
+                    ]
                 ),
             ]
         )
         try:
             shutil.copyfile(get_env_path(env, "requirements.txt"), requirements_path)
             subprocess.run(cmd, shell=True, check=True)
+        except subprocess.CalledProcessError as e:
+            subprocess.run(f"rm -rf {venv_path}", shell=True, check=True)
+            print("Partially built virtualenv for pyright environment master deleted.")
```

Comment:
hardcoded to "master"  but should vary right?

Reply:
doh